src/openxr/extensions/xr_fb_foveation_extension_wrapper.cpp

```diff
@@ -52,7 +52,7 @@ void XRFbFoveationExtensionWrapper::on_instance_destroyed() {
 	cleanup();
 }
 
-void **XRFbFoveationExtensionWrapper::set_swapchain_create_info_and_get_next_pointer(void **swapchain_create_info) {
+void **XRFbFoveationExtensionWrapper::set_swapchain_create_info_and_get_next_pointer(void **swapchain_create_info, bool p_depth_buffer) {
```

Comment:
@m4gr3d do we need to add the foveation extension when creating the depth buffer or only for the color buffer?

Reply:
Based on their documentation and their samples, it looks like it's only needed for the color buffer.
I'll double-check internally.

Reply:
Yes, it's only needed for the color buffer, so we can return `nullptr` when `p_depth_buffer` is `true`.